tests/test_symbols.py

```diff
@@ -39,260 +37,16 @@
     stack,
 )
 
+# Try to import utils normally. If it fails, assume that we are running the
+# tests from the inside the `tests/` directory to avoid importing
+# dwave.optimization locally.
+try:
+    from .utils import symbol_testing_utils
```

Comment:
if we do
```python
    from .utils import symbol_testing_utils as utils
```
we won't need to change the other lines. Avoiding some merge conflicts